R/pkg_assess.R

```diff
@@ -92,19 +92,39 @@ all_assessments <- function() {
 
 #' Get a specific set of assess_* functions for pkg_assess
 #'
-#' @param fxn_string vector of assess functions
+#' @param fxn_string vector of assess functionsto search for, can be a partial string
+#' @param invert logical. If TRUE then the assessment function(s) NOT in fxn_string will be returned, similar to grep -v
 #' @return a list of specific assess_* functions exported from riskmetric
 #'
 #' @importFrom utils packageName
 #' @export
-get_assessments <- function(fxn_string=""){
+get_assessments <- function(fxn_string="", invert=FALSE){
+  fs <- grep("^assess_[^.]*$",
+             getNamespaceExports(utils::packageName()),
+             value = TRUE)
+  fxn_string <- lapply(fxn_string, grep, x=fs, value=T)
+
+  if(length(fxn_string)==0 | all(sapply(fxn_string, length)==0)){
+    warning("Did not find any assessments that matched your query. Returning all assessments")
+    return(all_assessments())
```

Comment:
If I'm reading this right it will return all assessments if you pass a string that doesn't grep for any of the assessments. I'm thinking it might be better to just return nothing if fxn_string doesn't find anything. I'm thinking it might be unexpected behavior if we return something that wasn't actually found by group